src/helpers/isBoolean.ts

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
These helpers were created to avoid having to install lodash (which is heavy) but still having a way to use them whenever needed.